.gitignore

```diff
@@ -57,4 +57,3 @@ qtcreator-*
 CATKIN_IGNORE
 
 .fake
-
```

Comment:
This file currently shows there is a change even though nothing is actually different, so we can probably undo/revert/get rid of this change so that it no longer shows the file was changed